modules/core/src/shaderlib/project/viewport-uniforms.js

```diff
@@ -96,9 +96,13 @@ function calculateMatrixAndOffset({
       shaderCoordinateOrigin = [lng, lat];
     }
   }
+  if (shaderCoordinateSystem === PROJECT_COORDINATE_SYSTEM.IDENTITY) {
+    // we only support 2D 64-bit positions for now
```

Comment:
The use of 2D here is a bit unexpected. Maybe this is clearer? `// We only support 64-bit precision in the X and Y components of positions for now.`